package defpackage;
/* compiled from: r8-map-id-f559a6c9210f67746bd9ce87931ee85b676f3bda66e04dc5e85cb7e8eec71de1 */
/* renamed from: ic1  reason: default package */
/* loaded from: classes.dex */
public final class ic1 extends gc1 {
    public static final int b = Integer.numberOfLeadingZeros(31);
    public static final ic1 c = new gc1("CharMatcher.whitespace()");

    @Override // defpackage.ec1
    public final boolean a(char c2) {
        if ("\u2002\u3000\r\u0085\u200a\u2005\u2000\u3000\u2029\u000b\u3000\u2008\u2003\u205f\u3000\u1680\t \u2006\u2001  \f\u2009\u3000\u2004\u3000\u3000\u2028\n \u3000".charAt((48906 * c2) >>> b) == c2) {
            return true;
        }
        return false;
    }
}
